week1-recipe/styles.css

```diff
@@ -24,4 +24,16 @@ h3 {
   float: right;
   width: 600px;
   margin-left: 30px;
+}
+
+@media only screen and (min-width: 600px) {
+  [class *="left" ] {
```

Comment:
Instead of an attribute selector, you should be able to select this element with `.left`.